yarn.lock

```diff
@@ -2537,13 +2537,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@revenuecat/purchases-typescript-internal@npm:8.10.1":
```

Comment:
Hmm it's a bit weird this is removed and not updated?

Reply:
Oh! I will... undo because I agree

Reply:
OHHHHHH... it was because it was high level being required but I fixed the `react-native-purchases-ui/package.json` and its a dependency in there as well so it got moved under that one

I think this rooted because it was messed up in the parent bump branch 🤷‍♂️